core/modules/filter/js/filter.js

```diff
@@ -104,6 +104,39 @@ Backdrop.filterEditorDetach = function(field, format, trigger) {
   }
 };
 
+/**
+ * Provides summary text for the "Editor details" fieldset, under each CKEditor
+ * instance.
+ */
+Backdrop.behaviors.filterFieldsetSummaries = {
+  attach: function (context) {
+    $(context).find('fieldset.filter-wrapper').backdropSetSummary(function (element) {
+      var $element = $(element);
```

Comment:
Is this really necessary...?

Reply:
Yup. The fieldset breaks otherwise. Do you have a suggestion?

Reply:
Interesting. Wonder why...?

I haven't tested it, but I'd have thought that just replacing `$element` with `$(element)` in this function should work...

Reply:
...nah, scratch that; my bad. It's not needed indeed, and nothing breaks.

Reply:
...yes it does 😞 ...had to revert this.

Reply:
My js-fu is not that strong @BWPanda ...happy for you to push a change with whatever works, but removing this line causes the fieldset to break for me:

![image](https://user-images.githubusercontent.com/2423362/92335261-38ffab00-f09e-11ea-81aa-5f074da1c6d9.png)

Reply:
...also, see: https://github.com/backdrop/backdrop/blob/1.x/core/misc/collapse.js#L7